src/Network/Minio/Errors.hs

```diff
@@ -50,6 +50,12 @@ data ServiceErr = BucketAlreadyExists
 
 instance Exception ServiceErr
 
+isBucketAlreadyExists :: ServiceErr -> Bool
+isBucketAlreadyExists e = e == BucketAlreadyExists
```

Comment:
I don't think we need to add these functions to the library. It is very simple and does not add much benefit IMO.